File: Java/백준/Bronze/14470. 전자레인지/전자레인지.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] a = new int[5];
        int time =0;
        for (int i=0; i<5; i++){
            a[i] = in.nextInt();
        }
        while(a[0]<a[1]){
            if(a[0]<0){
                time+=a[2];
                a[0]++;
            }
            else if(a[0]==0){
                time+=a[3]+a[4];
                a[0]++;
            }
            else{
                time+=a[4];
                a[0]++;
            }
        }
        System.out.println(time);
    }
}

File: Python/백준/Bronze/14470. 전자레인지/전자레인지.py
a=[]
time=0
for i in range(5):
    a.append(int(input()))
while(a[0]<a[1]):
    if(a[0]<0):
        time+=a[2]
        a[0]+=1
    elif(a[0]==0):
        time+=a[3]+a[4]
        a[0]+=1
    else:
        time+=a[4]
        a[0]+=1
print(time)